mitosheet/src/mito/utils/actions.tsx

```diff
@@ -1375,12 +1392,16 @@ export const getActions = (
                 return doesColumnExist(startingColumnID, sheetIndex, sheetDataArray) ? defaultActionDisabledMessage : 'There are no columns in the dataframe to rename. Add data to the dataframe.'
             },
             searchTerms: ['rename', 'name', 'header'],
-            tooltip: "Rename the selected column."
+            tooltip: "Rename the selected column.",
+            displayKeyboardShortcuts: {
+                mac: 'Enter',
+                windows: 'Enter'
+            }
```

Comment:
I guess I was actually wrong about these -- oops. I think keyboard shortcuts are supposed to work from "anywhere" in the app. Enter only works if you have the column header selected. Sorry.